### PR TITLE
Analyze OMIM files

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/omim/issues/161

These are the analysis notebooks related to the discussion about disease defining gene associations to add into Mondo and changes in https://github.com/monarch-initiative/omim/pull/158. The initial analysis, v1, is the basis for the information in [OMIM Disease-Gene Issues](https://docs.google.com/document/d/1cLfBgPIZWiN5LX-E-xwSyBeFdT-vw0JuSfSM7HL3_hc/edit?tab=t.0#heading=h.h4y343h64cck).

@joeflack4 adding these notebooks here for reference. I'd like to keep the v1 (even though the working assumption of the rule was incorrect), since it was the work as the basis for the OMIM Disease-Gene Issues doc and helped find some other interesting cases in the data, e.g. gene association=1, mapping_key=3, but disease label contains 'digenic' and also the cases where the disease label is prefixed with [,{, or?

### CODE REVIEW

File: analyses/morbidmap-data-analysis/Analyze_morbidmap - v2.ipynb
Comment (joeflack4) [inline analyses/morbidmap-data-analysis/Analyze_morbidmap - v2.ipynb:1]: Looked through this and discussed on Slack. Basically didn't go through it with a magnifying glass, but from what I can see, the code is correct, reflecting the decisions made thus far, and the results should match with what is coming out of the pipeline latest updates:
- https://github.com/monarch-initiative/omim/pull/158

**Review by joeflack4 — APPROVED** [review]

**Review by matentzn — COMMENTED** [review]
I love it - generally this is how complicated tasks in ETL should be performed IMO, well done.

Like @joeflack4 I have not looked through the code with a fine comb, just, the results and things look good to me! Very interesting

**Comment by twhetzel** [comment]
Thanks @matentzn! The notebook analysis has been very helpful to identify nuances in the OMIM data and is what I've used as the basis for my questions to sort out the rules/conditions that should be used to get a list of 'disease defining' gene associations. It also helped QC the processing code Joe is working on taking only ~10 minutes to review/compare outputs from each of these.

**Comment by joeflack4** [comment]
@matentzn If you find these outputs worth looking at, feel free to look at this [google sheet](https://docs.google.com/spreadsheets/d/1-9V5nnsCc1rHXzzVrr9OuE9bTHTGYptT4QWP5kiTYRM/edit?gid=25340125#gid=25340125) as well, which basically has these same Disease->Gene outputs, plus also Gene->Disease, based on:
- #158

**Comment by twhetzel** [comment]
FYI - the gSheet is the result of the omim ingest/creation of the `omim.owl` file. Using Sparql against the `omim.owl` file, the gSheet content is created. The gSheet results should match what the notebook outputs related to diseases and their disease defining genes. The two are not just different views of the same exact file.

**Pull request merged**